src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -46,6 +46,14 @@ public void setFullView() {
         model.setFullView();
     }
 
+    public void setDayView() {
+        model.setDayView();
+    }
+
+    public void setListView() {
+        model.setListView();
+    }
```

Comment:
btw I know I added this first but idk if these are even necessary... like you can just do model.set() directly in the method LOL i was gg thru it! can change next time if need

Reply:
help i didnt even question it